okhttp/src/main/java/okhttp3/ResponseBody.java

```diff
@@ -103,7 +103,10 @@ public final Reader charStream() {
   /**
    * Returns the response as a string decoded with the charset of the Content-Type header. If that
    * header is either absent or lacks a charset, this will attempt to decode the response body as
-   * UTF-8.
+   * UTF-8. Closes {@link ResponseBody} automatically.
+   *
+   * <p>Warning: this method loads whole response body into memory and may cause
```

Comment:
Oh, got it, yeah my bad.

Rewritten it without word `Warning` and more as a thing to keep in mind, hope it's okay now and will help users.

Also, made a separate commit so you'll be able to use new cool GitHub features like ["Pick up where you left off"](https://github.com/blog/2123-more-code-review-tools) and ["Squash your commits"](https://github.com/blog/2141-squash-your-commits) :walking: